WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -44,7 +44,7 @@ extension NSNotification.Name {
     private var userName: String?
     private var userEmail: String?
     private var deviceID: String?
-    private var usingAnonymousIDForHelpCenter = false
+    private var haveUserIdentity = false
```

Comment:
Rather than adding another property, I'm just wondering if this could this be a computed `var` which checks whether `ZendeskUtils.sharedInstance.userEmail` and `ZendeskUtils.sharedInstance.userName` are not nil?

Reply:
I thought about that (and did try not to add a new property) , but it is possible for `userEmail` and `userName` to have values without the identity being created. It ended up being less convoluted by just using a Bool.